torchinductor/codegen/triton.py

```diff
@@ -47,14 +47,15 @@ def signature_of(arg):
 
 
 def config_of(args):
-    from triton.runtime.jit import JITFunction
+    from ..compile_fx import ALIGNMENT
+
+    def is_aligned(x):
+        if isinstance(x, TensorArg):
+            return x.buffer not in V.graph.unaligned_buffers
+        assert isinstance(x, SizeArg)
+        return V.graph.sizevars.maybe_guard_multiple_of(x.expr, ALIGNMENT)
 
-    divisible_by_16 = [
-        i
-        for i, arg in enumerate(args)
-        if isinstance(arg, TensorArg)
-        or V.graph.sizevars.maybe_guard_multiple_of(arg.expr, JITFunction.divisibility)
-    ]
+    divisible_by_16 = [i for i, arg in enumerate(args) if is_aligned(arg)]
```

Comment:
lol the place I worried about turned out to be problematic

Reply:
Yeah it did :)